src/ngraph/op/divide.cpp

```diff
@@ -78,6 +79,60 @@ shared_ptr<Node> ngraph::operator/(const Output<Node>& arg0, const Output<Node>&
     return make_shared<op::v0::Divide>(arg0, arg1);
 }
 
+namespace
+{
+    template <element::Type_t ET>
+    bool try_evaluate_divide(const EvaluatorTensorPtr& arg0,
+                             const EvaluatorTensorPtr& arg1,
+                             const EvaluatorTensorPtr& out,
+                             const op::AutoBroadcastSpec& broadcast_spec,
+                             bool pythondiv)
+    {
+        return (ET == arg0->get_element_type()) && (runtime::reference::divide(arg0->get_ptr<ET>(),
+                                                                               arg1->get_ptr<ET>(),
+                                                                               out->get_ptr<ET>(),
+                                                                               arg0->get_shape(),
+                                                                               arg1->get_shape(),
+                                                                               broadcast_spec,
+                                                                               pythondiv),
+                                                    true);
```

Comment:
I'm not a fan of tying the op definition so to a reference kernel. It seems to me that the op is just a definition of the op and that a separate class could be used to execute nodes thus separating the definition of the op from an particular implementation. Embedding an implementation here blurs the line between components.